_posts/2025-04-09-#31664.md

```diff
@@ -10,17 +10,96 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+### Motivation and Background
 
+The current Fee Estimator in Bitcoin Core ([`CBlockPolicyEstimator`](https://github.com/bitcoin/bitcoin/blob/4c1906a500cacab385b09e780b54271b0addaf4b/src/policy/fees.h#L148)) used by the wallet and exposed to clients via RPC, has two key limitations that affect users:
 
-## Questions
+1. **Mempool unaware**: Does not consider the current mempool unconfirmed transactions when estimating fees.
+
+2. **Package unaware**: Only considers individual transactions, ignoring parent-child relationships.
+
+These limitations cause practical problems for:
```

Comment:
You've briefly mentioned what the technical issues are, but haven't connected them to what the practical problems are.

package unaware -> can overestimate -> fees are overshot, wasting money
package unaware -> can underestimate -> people miss their confirmation targets
mempool unaware -> don't respond quickly enough to a rise in transaction volume -> underestimate -> people miss their confirmation targets

Reply:
Done